pages/tutorials/yaml/yaml-format.rst

```diff
@@ -0,0 +1,366 @@
+      <h1 class="display-3">YAML Format Tutorial</h1>
+
+   .. class:: lead
+
+      Here we describe the syntax and structure of Cantera YAML files, how
+      dimensional values in Cantera YAML files are handled, and how to
+      understand some of the error messages that may be encountered when reading
+      these input files.
+
+Syntax
+======
+
+Cantera YAML files use a subset of the `YAML 1.2
+<https://yaml.org/spec/1.2/spec.html>`__ specification. Cantera YAML files
+consist of individual values, which may be strings, numbers or booleans, that
+are then composed as elements of nested mappings and sequences.
+
+Strings
+-------
+
+Strings may be generally written without qoutes, but may be enclosed in single
+quotes or double quotes if needed in order to avoid certain parsing ambiguties.
+
+.. code:: yaml
+
+   A string
+   Another 'string'
+   "A string: that requires quotes"
+
+Numbers
+-------
+
+Numbers can be written as integers, decimal values, or using E-notation
+
+.. code:: yaml
+
+   3
+   3.14
+   6.022e23
+
+Booleans
+--------
+
+Boolean values in YAML are written as the words ``true`` or ``false``.
+
+Sequences
+---------
+
+A sequence of multiple items is specified by separating the items by commas and
+enclosing them in square brackets. The individual items can have any type --
+strings, integers, floating-point numbers (or even entries or other lists).
+
+.. code:: yaml
+
+   elements: [O, H, C, N, Ar]
+   temperature-ranges: [200.0, 1000.0, 3500.0]
+
+Sequences can also be written in "block" style, using one line for each item in
+the sequence, with each line starting with a dash:
+
+.. code:: yaml
+
+   elements:
+   - O
+   - H
+   - C
+   - N
+   - Ar
+
+Sequences can also be nested. The following examples are all equivalent:
+
+.. code:: yaml
+
+   data: [[1, 2], [3, 4]]
+
+   data:
+   -
+     - 1
+     - 2
+   -
+     - 3
+     - 4
+
+   data:
+   - - 1
+     - 2
+   - - 3
+     - 4
+
+Mappings
+--------
+
+A mapping is a container consisting of key--value pairs. The keys in a mapping
+must be unique. Like sequences, there are two ways to write a mapping. In the
+"flow" style, the mapping is enclosed in curly brackets, colons (followed by
+spaces) are used to separate keys and values, and key--value pairs are separated
+by commas:
+
+.. code:: yaml
+
+   composition: {H: 2, C: 1, O: 1}
+
+In the "block" style, each key is written on a new line, followed by a colon.
+The value can be placed either on the same line, or on the following line,
+indented one level:
+
+.. code:: yaml
+
+   composition:
+     H: 2
+     C:
+       1
+     O: 1
+
+All keys in Cantera YAML files are treated as strings. A Cantera YAML file is
+itself a mapping, usually in the block style. We refer to the keys in this
+top-level mapping as the "sections" of the input file.
+
+Sequences of Mappings
+---------------------
+
+A common structure in Cantera input files is a nested sequence of mappings. This
+can be written in the "block" style as:
```

Comment:
```suggestion
can be written in the block style as:
```

Reply:
I'm assuming this should be `**block style**` based on your other comments.